helm/values-uat.yaml

```diff
@@ -118,6 +118,7 @@ microservice-chart:
     TRANSACTION_CLOSE_PAYMENT_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-close-payment-queue
     TRANSACTION_REFUND_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-queue
     TRANSACTION_NOTIFICATIONS_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-queue
+    TRANSACTIONS_AUTHORIZATION_REQUESTED_QUEUE_NAME: pagopa-d-weu-ecommerce-transaction-auth-requested-queue
```

Comment:
please add parameter in canary delivery parameters with -b suffix, as done [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/0d4e622f9764bc2b5a91789f202095f40374e6fe/helm/values-uat.yaml#L25)